src/main/cli/settings.c

```diff
@@ -1352,25 +1352,25 @@ const clivalue_t valueTable[] = {
 #endif
 // PG_FLASH_CONFIG
 #ifdef USE_FLASH_CHIP
-    { "flash_spi_bus", VAR_UINT8 | MASTER_VALUE, .config.minmax = { 0, SPIDEV_COUNT }, PG_FLASH_CONFIG, offsetof(flashConfig_t, spiDevice) },
+    { "flash_spi_bus", VAR_UINT8 | MASTER_VALUE, .config.minmaxUnsigned = { 0, SPIDEV_COUNT }, PG_FLASH_CONFIG, offsetof(flashConfig_t, spiDevice) },
 #endif
 // RCDEVICE
 #ifdef USE_RCDEVICE
-    { "rcdevice_init_dev_attempts", VAR_UINT8 | MASTER_VALUE, .config.minmax = { 0, 10 }, PG_RCDEVICE_CONFIG, offsetof(rcdeviceConfig_t, initDeviceAttempts) },
-    { "rcdevice_init_dev_attempt_interval", VAR_UINT32 | MASTER_VALUE, .config.minmax = { 500, 5000 }, PG_RCDEVICE_CONFIG, offsetof(rcdeviceConfig_t, initDeviceAttemptInterval) },
+    { "rcdevice_init_dev_attempts", VAR_UINT8 | MASTER_VALUE, .config.minmaxUnsigned = { 0, 10 }, PG_RCDEVICE_CONFIG, offsetof(rcdeviceConfig_t, initDeviceAttempts) },
+    { "rcdevice_init_dev_attempt_interval", VAR_UINT32 | MASTER_VALUE, .config.u32Max = 5000, PG_RCDEVICE_CONFIG, offsetof(rcdeviceConfig_t, initDeviceAttemptInterval) },
```

Comment:
@azolyoung: I fixed this - the range type is determined at runtime based on the type of the parameter, so `minmax` does not work for `VAR_UINT32`:

```
# set rcdevice_init_dev_attempt_interval 
rcdevice_init_dev_attempt_interval = 3000
Allowed range: 0 - 327680500
Default value: 1000

# set rcdevice_init_dev_attempt_interval = 6000
rcdevice_init_dev_attempt_interval set to 6000
```

I recommend you properly fix this by turning the parameter into an `uint16_t` and reinstate the desired range.

Reply:
Ok, thank you!